src/main/java/org/ebml/ProtoType.java

```diff
@@ -48,6 +48,11 @@ public static Element getInstance(final ByteBuffer type)
   {
     final long codename = EBMLReader.parseEBMLCode(type);
     final ProtoType<? extends Element> eType = CLASS_MAP.get(Long.valueOf(codename));
+
+    if (eType == null) {
+      return null;
```

Comment:
I would also suggest returning Void to have the rest of the file getting parsed correctly and logging the unknown codename
```suggestion
      LOG.error("Unknown codename [" + codename + "]=[" + EBMLReader.bytesToHex(type.array()) + "], returning [Void]");
      return MatroskaDocTypes.Void.getInstance();
    
```